client/lib/plans/plans-list.js

```diff
@@ -680,7 +680,7 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'premium',
 		getDescription: () =>
 			i18n.translate(
-				'{{strong}}Best for Small Businesses:{{/strong}}' +
+				'{{strong}}Best for small businesses:{{/strong}}' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{strong}}Best for Small Businesses:{{/strong}}Comprehensive, automated scanning for security vulnerabilities, fast video hosting, and marketing automation.' )` *ES Score: 12*